client/plrdlg_common.cpp

```diff
@@ -436,13 +436,23 @@ QString col_research(const struct player *them)
     return _("-");
   } else if (BV_ISSET(them->client.visible, NI_TECHS)) {
     struct research *research = research_get(them);
-    return research_advance_name_translation(research,
-                                             research->researching);
+    if (research->client.researching_cost == 0) {
+      // Nothing is selected to research
+      return QStringLiteral("%1").arg(research_advance_name_translation(
```

Comment:
Maybe `%1 (%2/-)`? This way one sees how many bulbs are stored

Reply:
Ok I'll push another update. Good idea.